web/src/components/InlineTextFilter/InlineTextFilter.test.tsx

```diff
@@ -3,19 +3,20 @@ import { render, screen } from "@testing-library/react";
 import { InlineTextFilter } from "./InlineTextFilter";
 
 describe("InlineTextFilter", () => {
-	it("should render label with given text", () => {
-		render(<InlineTextFilter label="Test label" name="test" />);
-
-		const label = screen.getByText("Test label");
+	it("should render Input with given heading", () => {
```

Comment:
I think the name of this test should change to 'should render FilterGroup with given heading'.